images/win/scripts/Installers/Install-MongoDB.ps1

```diff
@@ -0,0 +1,9 @@
+####################################################################################
+##  File:  Install-MongoDB.ps1
+##  Desc:  Install MongoDB
+####################################################################################
+
+choco install mongodb
+
+Add-MachinePathItem "$($env:SystemDrive)\Program Files\MongoDB\Server\4.2\bin"
+$env:Path = Get-MachinePath
```

Comment:
We, actually, don't need this line because this line modifies Path only for current file and it doesn't make sense to do that in the last line

Reply:
fixed